osf/utils/machines.py

```diff
@@ -214,7 +214,8 @@ def notify_submit(self, ev):
         context = self.get_context()
         context['contributors_url'] = '{}contributors/'.format(self.machineable.target.absolute_url)
         context['project_settings_url'] = '{}settings/'.format(self.machineable.target.absolute_url)
-        for admin in self.machineable.target.contributors.filter(contributor__admin=True, contributor__node=self.machineable.target):
+
+        for admin in self.machineable.target.get_users_with_perm('admin'):
```

Comment:
Unsure about this one - should just traditional contributors get this email, or both trad contribs/group members?